### PR TITLE
Starting page

### PR DESCRIPTION
**Pull request by vicphan**
Changes
* added starting page
* added default fonts
* added theme extension
* added logo 
* added routes

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -7,34 +11,14 @@ import {
...
-          >
-            Learn Chakra
-          </Link>
-          <PlayerName />
-        </VStack>
-      </Grid>
-    </Box>
+    <Routes>
+        <Route path="/" element={<StartPage />}/>
+    </Routes>
```
Comment (sebcontreras) [inline src/App.tsx:22]: Nice, thanks for setting this up

File: src/pages/StartPage.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { Button, Flex, VStack } from '@chakra-ui/react';
+import  Logo from '../components/Logo';
+
+export default function StartPage(){
+    return (
+        <Flex backgroundImage='linear-gradient(to bottom right, green, yellow)' width='100%' height='calc(100vh)' alignItems='center' textAlign='center' justifyContent='center'>
```
Comment (sebcontreras) [inline src/pages/StartPage.tsx:6]: Love the gradient background

File: src/pages/StartPage.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+            <Flex alignItems='center'>
+                <VStack>
+                    <Logo size={["64px", "100px" ]} />
+                    <Button color='white' padding={8} fontSize={["large", '4xl']} colorScheme='whiteAlpha'>CREATE A NEW GAME</Button>
+                    <Button color='white' padding={8} fontSize={["large", '4xl']} colorScheme='whiteAlpha'>JOIN A GAME</Button>
+                </VStack>
+            </Flex>
+        </Flex>
```
Comment (sebcontreras) [inline src/pages/StartPage.tsx:14]: This is my first time using Chakra, looks very cool so far.

File: src/pages/StartPage.tsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import { Button, Flex, VStack } from '@chakra-ui/react';
+import  Logo from '../components/Logo';
+
+export default function StartPage(){
+    return (
+        <Flex backgroundImage='linear-gradient(to bottom right, green, yellow)' width='100%' height='calc(100vh)' alignItems='center' textAlign='center' justifyContent='center'>
+            <Flex alignItems='center'>
+                <VStack>
+                    <Logo size={["64px", "100px" ]} />
+                    <Button color='white' padding={8} fontSize={["large", '4xl']} colorScheme='whiteAlpha'>CREATE A NEW GAME</Button>
```
Comment (sebcontreras) [inline src/pages/StartPage.tsx:10]: I suggest making the buttons and text smaller. I think they are a little too close to the size of the logo, and they might be a little too large on mobile.

**Review by sebcontreras — APPROVED** [review]
Great work Vic, I like the theme that you used and it was cool seeing Chakra in action. Only one minor suggestion.

**Review by Jhappy77 — APPROVED** [review]

**Pull request merged**